client/src/templates/Challenges/components/bread-crumb.tsx

```diff
@@ -22,20 +25,24 @@ function BreadCrumb({ block, superBlock }: BreadCrumbProps): JSX.Element {
         <span className='ellipsis'>
           {i18next.t(`intro:${superBlock}.title`)}
         </span>
-      </Link>
-      <div className='breadcrumb-center' />
-      {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
-      {/* @ts-ignore */}
+      </Link>{' '}
```

Comment:
```suggestion
      </Link>
```
Nitpick: this doesn't seem to change anything.

Reply:
I did not make that change. So, I think the Linter wanted it, but will do some testing later.